etc/init.d/contextBroker.centos

```diff
@@ -91,7 +91,9 @@ broker_start()
 {
     printf "%-50s" "Starting ${NAME}..."
     if [[ -x ${EXECUTABLE} ]]; then
-        su $BROKER_USER -c "${EXECUTABLE} ${BROKER_OPTS} &> /dev/null "
+        # preserving existing error log (if it already exists)
+        su $BROKER_USER -C "mv ${BROKER_LOG_DIR}.${EXECUTABLE}.err ${BROKER_LOG_DIR}.${EXECUTABLE}.err.old 2> /dev/null || True"
```

Comment:
-C => -c   (Windows might not be case-sensitive as it is not a real operating system ;-) but Unix/Linux is)
True => true  (same same. /bin/true is an executable that returns ... true - this is to avoid a e.g. make to stop executing if something goes wrong false || true evaluates to true).

Suggestion: ${BROKER_LOG_DIR}.${EXECUTABLE} is very long.
Adding a variable 

    logFileBase=${BROKER_LOG_DIR}.${EXECUTABLE}

would make the line easier to read ...